markdownlint/TOP009_lessonOverviewItemsSentenceStructure/TOP009_lessonOverviewItemsSentenceStructure.js

```diff
@@ -0,0 +1,82 @@
+// A bullet list is every token between and inclusive bullet_list_open to bullet_list_closed
```

Comment:
I don't think we really need these comments in the file

Reply:
Oh yeah, forgot to remove them - They're misleading